sdk/grafana/Azure.ResourceManager.Grafana/CHANGELOG.md

```diff
@@ -1,14 +1,21 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0-beta.2 (2023-11-09)
```

Comment:
release date is already passed, need to set a future date

Reply:
Thanks, fixed to 2023-11-15